packages/@atjson/source-commonmark/src/index.ts

```diff
@@ -229,7 +229,7 @@ export default class CommonMarkSource extends Document {
     return {};
   }
 
-  toCommonSchema() {
+  toCommonSchema(): Document {
```

Comment:
question: Is this related to the work @tim-evans did here?
https://github.com/CondeNast-Copilot/copilot-atjson/pull/60/files#diff-7fa48f2ff3768ecdee9f0cafbf1d6633L5

Reply:
It is in the same area; I think it's better to go the other way and also add explicitness to source-copilot-markdown's toCommonSchema.

Reply:
Understood!